### DISCUSSION TITLE
Popular this week, this month, this year

### DISCUSSION THREAD

**Discussion by vanillajonathan** [asciinema/discussions#238] [Ideas]
When you explore recordings you can only sort by date or popularity. It would be great to be able to explore "popular this week", "popular this month" , and "popular this year".

Sites like Reddit have "Hot", "New and "Top". Under "Top" they have "Top Today", "Top This Week", "Top This Month", "Top This Year" and "Top All Time".

**Comment by ku1ik** [comment]
Yes! Great idea.

What's the difference between "hot" and "top"? I suppose we could define rules for popular/hot/top any way but I'm curious what metrics do they use.

**Reply by vanillajonathan** [reply]
I don't know, but here is an explanation by some guy.
https://www.reddit.com/r/modhelp/comments/2gt784/what_exactly_is_the_difference_between_hot_top/
